auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -69,24 +69,32 @@
             </tr>
           </thead>
         </template>
+
         <template v-slot:item="{ item, index }">
           <tr>
-            <td v-if="isNameRequest(item.corpType.code) && item.nameRequest" class="header col-wide">
+            <!-- Name Request Name(s) / Business Name -->
+            <td v-if="isNameRequest(item)" class="col-wide gray-9">
               <div v-for="(name, i) in item.nameRequest.names" :key="`nrName: ${i}`" class="pb-1 names-block">
                 <v-icon v-if="isRejectedName(name)" color="red" class="names-text pr-1" small>mdi-close</v-icon>
                 <v-icon v-if="isApprovedName(name)" color="green" class="names-text pr-1" small>mdi-check</v-icon>
-                <strong class="names-text">{{ name.name }}</strong><br>
+                <div class="names-text font-weight-bold">{{ name.name }}</div>
               </div>
             </td>
-            <td v-else class="header col-wide"><strong>{{ name(item) }}</strong></td>
+            <td v-else class="col-wide gray-9 font-weight-bold">{{ name(item) }}</td>
+
+            <!-- Number -->
             <td v-if="showCol(headers[1].text)">{{ number(item) }}</td>
-            <td v-if="showCol(headers[2].text)" class="type-col">
-              <span class="header"><strong>{{ type(item) }}</strong></span><br>
-              <span v-if="isNameRequest(item.corpType.code) && item.nameRequest">
-                {{ typeDescription(item.nameRequest.legalType) }}
-              </span>
+
+            <!-- Type -->
+            <td v-if="showCol(headers[2].text)" class="type-column">
+              <div class="gray-9 font-weight-bold">{{ type(item) }}</div>
+              <div>{{ typeDescription(item) }}</div>
```

Comment:
Looks fine.

![image](https://user-images.githubusercontent.com/10002889/200627586-7e1c13d9-edf5-4cd3-9d24-3b857c7ed69b.png)

Notes:
1. "Named Company" is a fallback value because the Auth API isn't yet telling the UI what type of registration this is. Eventually this will say "BC Sole Proprietorship" (for this example). This is one reason we shouldn't merge this PR yet.
2. Incorporation applications also show a sub-type (legal type).
3. Name requests now show a sub-type (legal type).
4. Businesses won't show a sub-type because the Type is already the legal type.
5. These changes have been approved by Yui and Scott.